operators/multicluster-operators-subscription/0.3.3/multicluster-operators-subscription.v0.3.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,813 @@
+                        matchExpressions:
+                        - key: app
+                          operator: In
+                          values:
+                          - multicluster-operators-application
+              containers:
+              - command:
+                - /usr/local/bin/multicluster-operators-placementrule
+                - --alsologtostderr
+                env:
+                - name: WATCH_NAMESPACE
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: DEPLOYMENT_LABEL
+                  value: multicluster-operators-application
+                - name: OPERATOR_NAME
+                  value: multicluster-operators-placementrule
+                livenessProbe:
+                  exec:
+                    command:
+                    - ls
+                  initialDelaySeconds: 30
+                  periodSeconds: 30
+                readinessProbe:
+                  exec:
+                    command:
+                    - ls
+                  initialDelaySeconds: 15
+                  periodSeconds: 15
+                image: quay.io/open-cluster-management/multicluster-operators-placementrule:2.4.0
+                imagePullPolicy: Always
+                name: multicluster-operators-placementrule
+                resources:
+                  limits:
+                    cpu: 1500m
+                    memory: 512Mi
+                  requests:
+                    cpu: 300m
+                    memory: 64Mi
+                volumeMounts:
+                - name: tmp
+                  mountPath: "/tmp"
+              - command:
+                - /usr/local/bin/gitopscluster
+                - --alsologtostderr
+                env:
+                - name: WATCH_NAMESPACE
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: DEPLOYMENT_LABEL
+                  value: multicluster-operators-application
+                - name: OPERATOR_NAME
+                  value: multicluster-operators-gitopscluster
+                livenessProbe:
+                  exec:
+                    command:
+                    - ls
+                  initialDelaySeconds: 15
+                  periodSeconds: 15
+                readinessProbe:
+                  exec:
+                    command:
+                    - ls
+                  initialDelaySeconds: 15
+                  periodSeconds: 15
+                image: quay.io/open-cluster-management/multicloud-integrations:latest
```

Comment:
It would be better to use 2.4.0 tag here.   I noticed there is no 2.4.0 tagged image published in the `quay.io/open-cluster-management/multicloud-integrations` yet. 

Also please rename the csv file to `multicluster-operators-subscription.v0.3.3.clusterserviceversion.yaml`  to align with the new version, though It doesn't hurt anything as the csv file name is not used for matching the csv version.